packages/web/src/components/Container/_Container.scss

```diff
@@ -0,0 +1,12 @@
+@use 'theme';
+@use 'tools';
+
+.Container {
+    width: 100%;
+    max-width: theme.$max-width;
+    margin-right: auto;
+    margin-left: auto;
```

Comment:
I think we can start using `-block` and `-inline` properties, they are supported well. What do you think?